package.json

```diff
@@ -76,12 +83,85 @@
     "isotope-layout": "3.0.x",
     "isotope-packery": "2.x.x",
     "lodash": "4.15.0",
-    "openlayers": "4.3.1",
-    "perfect-scrollbar": "0.7.x",
+    "ol": "^4.4.2",
+    "perfect-scrollbar": "^0.8.1",
+    "proj4": "2.3.3",
     "promise-queue": "2.2.3",
     "react": "15.3.1",
     "react-dom": "15.3.1",
-    "worldview-components": "^1.5.1",
-    "worldview-options-eosdis": "nasa-gibs/worldview-options-eosdis#v2.1.2"
+    "worldview-components": "^1.5.2",
+    "worldview-options-eosdis": "github:nasa-gibs/worldview-options-eosdis#v2.1.2"
+  },
+  "browser": {
```

Comment:
@Benjaki2 What's this browser key? How does it work? These dependencies should be installed in `dependencies` via npm, not in the `web/ext` folder.

Reply:
I found the documentation for the browser key: https://github.com/browserify/browserify#browser-field

But we should update this to use package names, and make sure that those packages are in the dependencies field.

Reply:
I did this here: https://github.com/nasa-gibs/worldview/pull/613